### PR TITLE
STORM-1154: SequenceFileBolt needs unit tests

### PR DESCRIPTION
**Pull request by dossett**
These are just a start, I am not terribly knowledgeable of SequenceFiles

### CODE REVIEW

**Comment by revans2** [comment]
The code looks good and is better then what we have now, which is nothing.  I am +1 on merging this in.

**Comment by harshach** [comment]
+1

File: external/storm-hdfs/src/test/java/org/apache/storm/hdfs/bolt/TestSequenceFileBolt.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
+package org.apache.storm.hdfs.bolt;
```
Comment (HeartSaVioR) [inline external/storm-hdfs/src/test/java/org/apache/storm/hdfs/bolt/TestSequenceFileBolt.java]: Please add Apache License header.

**Comment by HeartSaVioR** [comment]
Except minor header issue then it's great! +1.

**Comment by dossett** [comment]
Thanks, everyone! @HeartSaVioR I added the Apache License Header.

Would adding something like the Apache Rat plugin to the build process be helpful to audit issues like this?  I saw at least one other file I contributed recently is also missing the Apache License.  I would be happy to create a JIRA to use Rat.

**Comment by revans2** [comment]
@dossett Yes please do look into it.  I know we use it for releases so I think it is already configured in the pom.xml, but if we could make it is a part of the default build that would catch problems like this sooner, rather then later.

**Comment by dossett** [comment]
@revans2 Opened STORM-1161 and submitted a PR to do just that.

**Comment by revans2** [comment]
Still +1

**Pull request merged**